### PR TITLE
Images Try 1

### PR DESCRIPTION
**Pull request by elisenharper**

### CODE REVIEW

**Comment by elisenharper** [comment]
I'm pretty stumped on this one. I followed along in the book and then did some Google searching and still can't seem to get it to work. When I run it, it acts like it's adding and resizing the photos but it doesn't save them to the new file.

**Comment by elisenharper** [comment]
I FINALLY got it to work on my computer but I seem to be having problems uploading here

File: resizeAndAddLogo.py [hunk 1]
```diff
@@ -8,7 +10,40 @@
...
 #TODO: Calculate the new width and height to resize to.
+        if width > height:
+            height = int((SQUARE_FIT_SIZE / width) * height)
+            width = SQUARE_FIT_SIZE
+        else:
+            width = int((SQUARE_FIT_SIZE / height) * width)
+            height = SQUARE_FIT_SIZE
+
+#Resize the image.
+    print('Resizing %s...' %(filename))
```
Comment (halfaro1) [inline resizeAndAddLogo.py]: @elisenharper shouldn't this be indented so it only resizes if it falls into the previous `if` block?

**Comment by elisenharper** [comment]
Worked on indenting the resize block and it still failed. It's a good thing these pictures are so cute because I'm getting frustrated!

**Comment by halfaro1** [comment]
@elisenharper 
The automated testing for this particular assignment isn't working as expected, so I've graded your project manually. Everything is 👍 on your project, so I am closing this and changing the tag to `graded`. When I transfer these grades to Blackboard, you'll see the tag switch to `grade updated on blackboard`. Great job!

**Pull request closed without merging**